src/main/resources/logback.xml

```diff
@@ -1,10 +1,8 @@
 <configuration>
 
-    <conversionRule conversionWord="coloredLevel" converterClass="play.api.libs.logback.ColoredLevel" />
-
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
         <encoder>
-            <pattern>%coloredLevel - %logger - %message%n%xException</pattern>
+            <pattern>%level - %logger - %message%n%xException</pattern>
```

Comment:
i think this log config was originally copied from one of our play framework projects. this change avoids warnings about `ColoredLevel` not being on the classpath